content/articles/managing-docker-containers-using-command-line-interface/index.md

```diff
@@ -0,0 +1,97 @@
+### Introduction
+
+With Docker, you can build, test, and deploy applications in the form of portable containers that can be used nearly anywhere. When using Docker, you might quickly get a huge number of useless items, which take up a lot of disk space and clog up the Docker command output. Unless you specifically instruct Docker too, it will not delete unneeded items like containers, images, volumes, and networks. Some of the most effective techniques to manage Docker Containers directly from the Command Line Interface are discussed in this article. We will look at some real-world examples of the most often used but very effective and efficient Docker commands that will make your Docker system organized and save up disk space by deleting unneeded Docker containers, images, volumes, and networks.
+
+### Table of contents
+
+- [Using interactive shell to run a docker container](#using-interactive-shell-to-run-a-docker-container)
+- [Remove all the dangling volumes](#remove-all-the-dangling-volumes)
+- [Removing docker containers and images](#removing-docker-containers-and-images)
+- [Using aliases](#using-aliases)
+- [Examining Docker containers](#examining-docker-containers)
+- [Conclusion](#conclusion)
+
+### Using interactive shell to run a docker container
+
+In most circumstances, an `interactive shell` reads and writes to the user's terminal. Interactive behavior is enabled when the bash command is used without any non-option arguments, unless the option is a text to read from or the shell is started to read from standard input, in which case positional parameters can be provided.

```

Comment:
This sentence, "Interactive behavior........." is too long. Consider breaking it into two parts.